cmake/tribits/doc/developers_guide/TribitsDevelopersGuide.rst

```diff
@@ -6577,6 +6577,29 @@ install commands from the former two files are run before install commands for
 the enabled packages while install commands from the latter two files are run
 after.)
 
+One can also change what compilers are written into the generated
+``<Project>Config.cmake`` and ``<Package>Config.cmake`` files for the build
+and the install trees.  By default, the compilers pointed to in these
+``Config.cmake`` files will be ``CMAKE_<LANG>_COMPILER`` where ``<LANG>`` =
+``CXX``, ``C``, and ``Fortran``.  But if can change this by setting any of the
```

Comment:
typo: "But if" - > "But one"

Reply:
Typo is fixed in https://github.com/TriBITSPub/TriBITS/commit/f6c9f1ae9065dedc576d6890ec95aa1466a1815b.